docs/Set-PSReadLineKeyHandler.md

```diff
@@ -1,150 +1,157 @@
 ---
+external help file: Microsoft.PowerShell.PSReadLine.dll-Help.xml
+keywords: powershell,cmdlet
+locale: en-us
+Module Name: PSReadLine
+ms.date: 12/07/2018
+online version: http://go.microsoft.com/fwlink/?LinkID=821452
 schema: 2.0.0
-external help file: Microsoft.PowerShell.PSReadLine2.dll-help.xml
+title: Set-PSReadLineKeyHandler
 ---
 
 # Set-PSReadLineKeyHandler
 
 ## SYNOPSIS
-
-Binds or rebinds keys to user defined or PSReadLine provided key handlers.
+Binds keys to user-defined or PSReadLine key handler functions.
 
 ## SYNTAX
 
-### Set 1
+### ScriptBlock
 
 ```
-Set-PSReadLineKeyHandler [-Chord] <String[]> [-ScriptBlock] <ScriptBlock> [-BriefDescription <String>]
- [-Description <String>] [-ViMode <ViMode>]
+Set-PSReadLineKeyHandler [-ScriptBlock] <ScriptBlock> [-BriefDescription <String>] [-Description <String>]
+ [-Chord] <String[]> [-ViMode <ViMode>] [<CommonParameters>]
 ```
 
-### Set 2
+### Function
 
 ```
-Set-PSReadLineKeyHandler [-Chord] <String[]> [-Function] <String> [-ViMode <ViMode>]
+Set-PSReadLineKeyHandler [-Chord] <String[]> [-ViMode <ViMode>] [-Function] <String> [<CommonParameters>]
 ```
 
 ## DESCRIPTION
 
-This cmdlet is used to customize what happens when a particular key or sequence of keys is pressed while PSReadLine is reading input.
-
-With user defined key bindings, you can do nearly anything that is possible from a PowerShell script.
-Typically you might just edit the command line in some novel way, but because the handlers are just PowerShell scripts, you can do interesting things like change directories, launch programs, etc.
+The `Set-PSReadLineKeyHandler` cmdlet customizes the result when a key or sequence of keys is
+pressed. With user-defined key bindings, you can do almost anything that is possible from within a
+PowerShell script.
 
 ## EXAMPLES
 
-### --------------  Example 1  --------------
+### Example 1: Bind the arrow key to a function
 
-```
-PS C:\> Set-PSReadLineKeyHandler -Key UpArrow -Function HistorySearchBackward
+This command binds the up arrow key to the function **HistorySearchBackward**. This function uses
+the current contents of the command line as the search string used to search the command history.
```

Comment:
This function searches command history for command lines that **start with** the current contents of the command line.